contracts/modules/oracle/Oracle.sol

```diff
@@ -70,4 +70,37 @@ contract Oracle is IOracle {
             return NULL_PRICE;
         }
     }
+
+    function _getLatestAnswer(address token) external view returns (int256) {
```

Comment:
Add comment "view function for use offchain. Can be used onchain for less gas than `getLatestAnswer` with security tradeoffs"

Reply:
done